README.md

```diff
@@ -320,6 +320,35 @@ await service.initializePayment({
 });
 ```
 
+Also, [PayPal](https://support.bigcommerce.com/s/article/Connecting-with-PayPal) (also known as PayPal Commerce Platform) requires specific options to initialize the PayPal Smart Payment Button on checkout page that substitutes a standard submit button
+
+```html
+<!-- This is where the PayPal button will be inserted -->
+<div id="container"></div>
+```
+
+```js
+service.initializePayment({
+    methodId: 'paypalcommerce',
+    paypalcommerce: {
+        container: '#container',
+        submitForm: () => {
+            service.submitOrder({ methodId: 'paypalcommerce', });
```

Comment:
@helga1507 payload should be 
```js
    payment: {
        methodId: 'paypalcommerce',
    },
```

Reply:
Can you update it here as well 
https://github.com/bigcommerce/checkout-sdk-js/blob/master/docs/interfaces/paypalcommercepaymentinitializeoptions.md